buying_selling/config/settings.py

```diff
@@ -113,8 +113,8 @@
 SIGNING_KEY = env("SIGNING_KEY", default="",)
 
 SIMPLE_JWT = {
-    'ACCESS_TOKEN_LIFETIME': timedelta(minutes=5),
-    'REFRESH_TOKEN_LIFETIME': timedelta(days=1),
+    'ACCESS_TOKEN_LIFETIME': timedelta(days=1),
+    'REFRESH_TOKEN_LIFETIME': timedelta(days=2),
```

Comment:
Increased them for now, we will revoke these changes before production

Reply:
This was very much required, improves productivity